bundle/bundle_test.go

```diff
@@ -949,3 +949,40 @@ func TestBundle_IsOutputSensitive(t *testing.T) {
 	})
 
 }
+
+func TestBundle_GetAction(t *testing.T) {
+	testcases := []struct {
+		action    string
+		modifies  bool
+		stateless bool
+		wantErr   string
+	}{
+		{action: "install", modifies: true},
+		{action: "install", modifies: true},
```

Comment:
Perhaps cases 2 and 3 were meant to have `action` as "upgrade" and "uninstall" ?

Reply:
I just really wanted to test install like _extra_ hard... 🤦‍♀️

Reply:
Fixed!